Sources/ProjectDescription/Dependency.swift

```diff
@@ -0,0 +1,35 @@
+//
```

Comment:
Nitpick: I'd remove the headers from all the files to be consistent with the rest of the project.